### DISCUSSION TITLE
[Flatpak] Enable Flathub on fresh install.

### DISCUSSION THREAD

**Discussion by darkdragon-001** [elementary/appcenter#1452] [Ideas]
## What Happened
Fresh install of Elementry OS 6 Daily.

See _Steps to Reproduce_.

## Expected Behavior
All options listed above should work out of the box immediately.

## Steps to Reproduce

1. Search in App Center for flathub apps: no results
2. Try to install via Terminal (`flatpak install flathub org.videolan.VLC`): no results
3. Install via flathub.org website + sideload: works  
   _(This also installes new user remote flathub which is not filtered like the default system one!)_
4. Install additional Flatpak app via Terminal: works now
5. Check for apps in _Applications_ menu: none is shown
6. Reboot
7. Check for apps in _Applications_ menu: all listed now immediately

## Platform Information
Elementary OS 6 Daily.

**Comment by cassidyjames** [comment]
Please be sure to search both open and closed issues before filling a new one. See https://github.com/elementary/appcenter/issues/1332 for prior discussion on this topic.

I do think it is worth keeping an issue like this open for consideration/discussion, especially as it is a relatively common request; however, it is not the responsibility of AppCenter to determine included remotes and repositories. As such, I will move this to the OS repo where default remotes is currently handled.

**Comment by danirabbit** [comment]
This is done now! So closing as completed :)
